### PR TITLE
Fix requirements docs txt

### PR DESCRIPTION
**Pull request by pacrob**
## What was wrong?

readthedocs was building with the wrong versions, related to https://github.com/ethereum/ethereum-python-project-template/issues/40

## How was it fixed?

Discovered a `.readthedocs.yml` format that renders `requirements-docs.txt` unnecessary.

Removed `requirements-docs.txt`, moved it all to `.readthedocs.yml`.

Also, prefer `.yml` to `.yaml`.

### To-Do

[//]: # (Stay ahead of things, add list items here!)
- [x] Clean up commit history

[//]: # (For important changes that should go into the release notes please add a newsfragment file as explained here: https://github.com/ethereum/eth-abi/blob/master/newsfragments/README.md)

[//]: # (See: https://eth-abi.readthedocs.io/en/latest/contributing.html#pull-requests)
- [x] Add entry to the [release notes](https://github.com/ethereum/eth-abi/blob/master/newsfragments/README.md)

#### Cute Animal Picture

image

### CODE REVIEW

File: newsfragments/204.internal.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Moved requirements-docs.txt info info .readthedocs.yml
```
Comment (fselmo) [inline newsfragments/204.internal.rst]:
```suggestion
Moved requirements-docs.txt info into .readthedocs.yml
```

File: .readthedocs.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+version: 2
```
Comment (fselmo) [inline .readthedocs.yml:1]: This looks like it differs from the template repo... what's the end result for this setup across all projects? We should maybe start there.

edit: I imagine it's this setup... and that lgtm... we should add it to the template repo if so.

**Review by fselmo — APPROVED** [review]
Ah, I missed the comment links... I see the context. This lgtm as the tester 👌🏼 

So, just the typo in the newsfragment :)

File: newsfragments/204.internal.rst (hunk 1 above)
Comment (pacrob) [reply newsfragments/204.internal.rst]: thanks for catching!

**Pull request merged**